webpack/assets/javascripts/bundle_novnc.js

```diff
@@ -52,20 +53,22 @@ $(document).on('ContentLoad', () => {
   const vncScreen = $('#noVNC_screen');
 
   if (vncScreen.length) {
-    $('#sendCtrlAltDelButton').on('click', sendCtrlAltDel);
 
+    $('#sendCtrlAltDelButton').on('click', sendCtrlAltDel);
     const protocol = $('#vnc').data('encrypt') ? 'wss' : 'ws';
-    const host = window.location.hostname;
-    const port = $('#vnc').attr('data-port');
-    const url = `${protocol}://${host}:${port}`;
+    const host = $('#vnc').attr('data-host') || window.location.hostname;
+    const port = $('#vnc').attr('data-port') || $('#vnc').attr('data-proxy_port');
```

Comment:
Looks like a leftover from a previous iteration:
```suggestion
    const port = $('#vnc').attr('data-port');
```